src/Components/DashBoard/Repository/RepoComponents/RepoDetails/RepoDetailBackdrop/RemoteConfigComponent/RemoteCard.js

```diff
@@ -101,6 +101,33 @@ export default function RemoteCard(props) {
 
     return remoteLogo;
   };
+
+  const remoteUrlHandler = (gitRemoteData) => {
+    let remoteData = "";
+    if (gitRemoteData) {
+      if (
+        gitRemoteData.match(/(^https)/gi) ||
```

Comment:
Follow the review comments posted for [RepoLeftPaneComponent.js](src/Components/DashBoard/Repository/RepoComponents/RepoDetails/RepoLeftPaneComponent.js)

And lets do the URL re-direct for `https://` alone and exclude it for `http` for security reasons